### DISCUSSION TITLE
Run a test isolated (prevent entire suite to run)

### DISCUSSION THREAD

**Discussion by Warkanlock** [nodejs/help#4951] [Q&A]
Hi everyone, new here so question might be a little bit silly,

But, is there a way to run a test isolated just to not run the entire suite all at once?

Regards

**Comment by Trott** [comment]
If you mean Node.js core tests, how to run a single test is documented in https://github.com/nodejs/node/blob/main/BUILDING.md#running-tests. If you mean in a Node.js project in general, it will depend on that project's tooling.
